src/components/exercises/ShowLetter.js

```diff
@@ -26,26 +26,7 @@ const styles = {
   }
 };
 
-const UnwrappedShowLetter = ({
-  letter,
-  sounds: [letterSound, repeatSound],
-  isRepeat
-}) => {
-  const play = () => {
-    if (isRepeat) this.icon.unfocus();
-    letterSound.playAsync();
-    letterSound.setPlaybackFinishedCallback(() => {
-      letterSound.setPositionAsync(0);
-      if (isRepeat) {
-        this.icon.focus();
```

Comment:
This effect is missing in the new version, we probably need an callback for this in the playAll helper function.